gr-blocks/lib/wavfile_source_impl.h

```diff
@@ -15,6 +15,9 @@
 #include <gnuradio/blocks/wavfile_source.h>
 #include <sndfile.h> // for SNDFILE
 
+#define ITEMS_SIZE 1024
```

Comment:
If these are by definition the same value as in sink, maybe find a single place for to specify them?

Or are these abviously forever the same value always for wav files per spec?

Reply:
Definitely could be different. In fact, I'm going to change the sink to 8192 to replicate the previous behavior.